apps/public-docsite-v9/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33270&commitId=79660870a468bd1664dd57a1402cef7ba45827c8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|40| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3320| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2561| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>